apps/storefront/app/grunnleggende/for-utviklere/fargar/page.mdx

```diff
@@ -0,0 +1,140 @@
+import { CodeIcon } from '@navikt/aksel-icons';
+
+import { MenuPageLayout } from '@layouts';
+import { CodeSnippet } from '@repo/components';
+
+export const metadata = {
+  title: 'Fargar',
+  description: 'Informasjon om korleis du bruker fargar på frontend.',
+};
+
+export default ({ children }) => (
+  <MenuPageLayout
+    content={children}
+    data={{
+      title: 'Fargar',
+      date: '26. november 2024',
+      icon: <CodeIcon />,
+      color: 'blue',
+    }}
+  />
+);
+
+Denne artikkelen føreset at du allereie har definert fargane du skal bruke ved hjelp av [Temabyggeren](/grunnleggende/for-designere/eget-tema).
+
+HTML data-attributt vert brukt for å definere farge på ein komponent. Komponentane kan nytte alle fargane i temaet.
+
+Når du skal setje farge på ein komponent, er det to data-attributt som kan nyttast:
+- `data-color-scheme` som endrar fargemodus
+- `data-color` som endrar farge
+
+## `data-color-scheme`
+
+Dette data-attributtet blir brukt for å veksle mellom lys, mørk eller andre fargemodusar. Innhald som er barn av dette data-attributtet 
+vil få modusen spesifisert av forelder. Som standard vil lys modus bli nytta.
+
+<CodeSnippet language="html">
+{`
+<body>
+  <div>
+    <!-- Innhaldet her vil ha lys fargemodus -->
+  </div>
+
+  <div data-color-scheme="dark">
+    <!-- Innhaldet her vil ha mørk fargemodus -->
+  </div>
+</body>
+`}
+</CodeSnippet>
+
+Du kan endre `data-color-scheme` på heile eller delar av sida og byte mellom ulike fargemodus så ofte du vil. Til dømes kan hovudsida vere lys, 
+men med ei mørk botnlinje (footer). Du kan då setje `data-color-scheme="dark"` på botnlinja.
+`data-color-scheme` kan bli satt til `light`, `dark` eller `auto`. `auto` vil følgje systeminnstillingane til brukaren.
```

Comment:
Foreslår å splittet denne opp i to avsnitt? Snakker om to ting her. Ble knotet å lese. 

Kanskje `data-color-scheme` og den verdier som et eget avsnitt 

<img width="764" alt="image" src="https://github.com/user-attachments/assets/588ec3eb-7a63-414c-8d4c-20c5c996f505" />


```suggestion
Du kan endre `data-color-scheme` på heile eller delar av sida og byte mellom ulike fargemodus så ofte du vil. Til dømes kan hovudsida vere lys, 
men med ei mørk botnlinje (footer). Du kan då setje `data-color-scheme="dark"` på botnlinja.

`data-color-scheme` kan bli satt til `light`, `dark` eller `auto`. `auto` vil følgje systeminnstillingane til brukaren.
```